source/documentation/common/upgrade/proc-Upgrading_the_Manager_to_4-4.adoc

```diff
@@ -178,12 +178,12 @@ endif::rhv-doc[]
 +
 [source,terminal,subs="normal"]
 ----
-# yum install ovirt-engine-extension-aaa-ldap ovirt-engine-extension-aaa-misc ovirt-engine-extension-logger-log4j
```

Comment:
given the fact that virtually everyone is using aaa right now or in previous versions we should keep this, as this section talks about upgrades from 4.3 and such.
I think log4j can be removed though, correct, @mwperina ?

Reply:
- Yes, log4j was completely removed...
- I'll retain the aaa-ldap in the given command
- and add the deprecation notice

Reply:
That upgrade part depends what was configured in previous version:

1. If customers user aaa-ldap, then they will need to install
    yum install ovirt-engine-extension-aaa-ldap ovirt-engine-extension-aaa-misc
2. If customers already switched to RH SSO, then they will need to install only 
    yum install ovirt-engine-extension-aaa-misc